benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -126,6 +127,21 @@ def split_list(arr, n):
     return [arr[i::n] for i in range(n)]
 
 
+def get_ps_metrics(config, ps_rrefs):
+    rank_metrics = []
+    if config.ps_count > 0:
+        ps_config = config.ps_config
+        ps = get_benchmark_ps_map()[str(ps_config["ps_class"])]
+        for rank, ps_rref in ps_rrefs.items():
+            metrics = rpc.rpc_sync(
+                ps_rref.owner(),
+                ps.get_metrics_rpc,
+                args=(ps_rref,)
+            )
```

Comment:
can this be simplified to the following and get rid of the `ps` arg?

```python
ps_rref.rpc_sync().get_metrics_rpc()
```

Reply:
yes simplified call and removed `ps` variable.